lib/index.js

```diff
@@ -191,7 +191,7 @@ export async function verifyCapabilityInvocation({
   });
   // invocation target must match absolute url
   let invocationTarget;
-  if(url.includes(':')) {
+  if(url.startsWith('https://')) {
```

Comment:
This would disallow other protocols, but I think that's largely untested anyway. I'll file an issue to add support if needed for that.